sdk/Trace/NoopSpan.php

```diff
@@ -13,7 +13,7 @@ class NoopSpan implements API\Span
 {
     use ContextValueTrait;
 
-    /** @var API\SpanContext */
+    /** @var API\Baggage */
     private $context;
```

Comment:
Just a nit, wondering if this should also be changed to `baggage` for clarity down the road (for those looking later who don't know the history of the change).

Reply:
I think this is prudent.

Reply:
Thinking about this statement more, there is a difference between `Context` and SpanContext`.  Perhaps it's smart to just change `SpanContext`?

Reply:
I'm okay with this staying as is given how our implementation implements them both;   although I still think changing the internal variables might add more clarity, but this is just a nit and not merge blocking.  The [spec](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/baggage/api.md) allows for more distinct entities, but php baggage and context seem a lot more tightly coupled at this point.